imgui/__init__.pyi

```diff
@@ -1163,6 +1163,17 @@ def plot_lines(
     values_offset: int = 0,
     stride: int = ...,
 ): ...
+def plot_histogram(
+    label: str,
+    values: array[float] | Any,  # https://github.com/cython/cython/issues/6272
```

Comment:
[Cython memory view types](http://docs.cython.org/en/latest/src/userguide/memoryviews.html) cannot be used with standard python type checking notation: https://github.com/cython/cython/issues/6272